packages/rules-unit-testing/package.json

```diff
@@ -32,8 +33,11 @@
     "rollup": "2.7.6",
     "rollup-plugin-typescript2": "0.27.0"
   },
+  "peerDependencies": {
+    "firebase-admin": "^8.13.0"
```

Comment:
This will need to be updated when https://github.com/firebase/firebase-js-sdk/pull/3228 is released.